x-pack/plugins/spaces/public/share_saved_objects_to_space/components/selectable_spaces_control.tsx

```diff
@@ -191,7 +200,12 @@ export const SelectableSpacesControl = (props: Props) => {
       <EuiFormRow
         label={selectSpacesLabel}
         labelAppend={
-          <EuiFlexGroup direction="column" gutterSize="none" alignItems="flexEnd">
+          <EuiFlexGroup
+            direction="column"
```

Comment:
I would keep these in a row layout as it shifts oddly when the `hidden` count is shown.
<img width="592" alt="Screen Shot 2022-03-30 at 15 03 13 PM" src="https://user-images.githubusercontent.com/549577/160911428-6232a844-8153-465e-9e3f-98e842940e57.png">

The other option is to actually combine this count with the additional message that was added to the bottom:
<img width="592" alt="Screen Shot 2022-03-30 at 15 03 13 PM" src="https://user-images.githubusercontent.com/549577/160911630-266db1d3-707d-420d-be0d-152a9e264802.png">

Reply:
Good idea, I went with your second recommendation, see 832472282cc1d5ad4ae8a9811495d67036447977